lisp/i18n/ts/fr_FR/lisp.ts

```diff
@@ -216,17 +216,17 @@
     <message>
       <location filename="../../../plugins/cart_layout/layout.py" line="190"/>
       <source>Countdown mode</source>
-      <translation type="unfinished">Countdown mode</translation>
+      <translation>Mode de compte à rebours</translation>
     </message>
     <message>
       <location filename="../../../plugins/cart_layout/layout.py" line="193"/>
       <source>Show seek-bars</source>
-      <translation type="unfinished">Show seek-bars</translation>
+      <translation>Afficher les barres de recherche</translation>
     </message>
     <message>
       <location filename="../../../plugins/cart_layout/layout.py" line="194"/>
       <source>Show dB-meters</source>
-      <translation type="unfinished">Show dB-meters</translation>
+      <translation>Afficher le niveau de décibels</translation>
```

Comment:
There is (at least) another translation of the exact same sentence above. I very much prefer this one.